src/components/RefinementList/__tests__/RefinementList-test.js

```diff
@@ -332,18 +333,18 @@ describe('RefinementList', () => {
         searchFacetValues: x => x,
         templateProps: {
           templates: {
-            item: item => item,
-            searchableNoResults: x => x,
+            item: item => item.value,
+            searchableNoResults: x => JSON.stringify(x),
```

Comment:
what are these two changes used for?

Reply:
Testing Library stringifies objects to `[object Object]`, Enzyme's behavior was closer to `JSON.stringify`. Calling `JSON.stringify` tracks what parameters are passed to the templates.